src/components/search_bar/search_bar.tsx

```diff
@@ -37,7 +37,7 @@ import { Query } from './query';
 import { CommonProps } from '../common';
 import { OuiFieldSearchProps } from '../form/field_search';
 
-export { Query, AST as Ast } from './query';
```

Comment:
I think changing `Ast` -> `AST` would be a breaking change. Would a separate export directly from the original file work?

Reply:
Thank you for bringing this up! I pushed a new commit reverting my changes just so I can preserve the commit history. I've tried to research this issue further, but I think I may be heading down a rabbit hole. :) Would you suggest tabling this issue for right now? Or do you have any other suggestions for a next step?

Reply:
Sure, feel free to focus on something else. If it's not blocking anything then it doesn't need to be a priority. I would recommend converting it to draft if you're not actively working on it.

Reply:
Sounds good. No, it's not blocking anything. I'll convert this PR to a draft.

Reply:
I've recently updated my MacOS to 14.0 (Sonoma). I don't know if that affected anything, but I'm no longer getting the warning in my console about the `Ast` module not being found when I run the most recent version of the `oui` repo. Do you think that it's fine to go ahead and close this PR, as the issue seems to be resolved?

Reply:
Yeah I think you could close it in that case

Reply:
Sounds good. I'll close it.